src/Components/Search/Suggestions/SuggestionItem.tsx

```diff
@@ -113,36 +119,68 @@ const QuickNavigation: React.FC<{
   showArtworksButton: boolean
   showAuctionResultsButton: boolean
 }> = ({ href, showArtworksButton, showAuctionResultsButton }) => {
+  const { trackEvent } = useTracking()
+
+  const handleQuickNavigationClick = (type: "artworks" | "auction-results") => {
+    const label = type === "artworks" ? "Artworks" : "Auction Results"
+    const destinationPath =
+      type === "artworks" ? "/works-for-sale" : "/auction-results"
+
+    const trackingEvent: SearchSuggestionQuickNavigationItemSelected = {
+      context_module: ContextModule.header,
+      destination_path: href + destinationPath,
+      action: ActionType.searchSuggestionQuickNavigationItemSelected,
```

Comment:
suggestion: The name of the event has changed since yesterday's update. Should be "selectedSearchSuggestionQuickNavigationItem", I believe.

Reply:
perfect! Will update when https://github.com/artsy/force/pull/12039 passes checks and merged